bin/server.py

```diff
@@ -145,13 +147,17 @@ async def change_root_password(self, request, password):
 
     @only_if_initialized
     @remote
-    async def sync_configuration(self, request, image_configuration_params):
-        request.ret(READY)
+    async def get_configuration_params(self, request):
+        request.ret(self._configuration)
 
     @only_if_initialized
     @remote
-    async def get_default_configuration(self, request):
-        request.ret(defaults.CONFIGURATION)
+    async def set_configuration_params(self, request, configuration):
```

Comment:
The class attribute is named `_configuration`, not "configuration_params". Therefore, it is logical to call the procedures `get_configuration`/`set_configuration` and not to arrange a zoo.